staff/mgarciarojas/Javascript exercises/multiplicationTableOnDemand.js

```diff
@@ -0,0 +1,9 @@
+// Multiplications table on demand  
+//Write a function that writes in the console the multiplication table (in one column) of any number passed as parameter
+
+function multiplicationTableOnDemand(RequestedNumber){
+    for (i=1; i<=10; i++){
+        console.log(RequestedNumber*i)
```

Comment:
Use `console.log` as a complementaty thing not as the main logic of your `function`. Use `return` instead

Reply:
Done